NEWS

```diff
@@ -1,5 +1,15 @@
 systemd System and Service Manager
 
+CHANGES WITH 248:
+
+        * The MountAPIVFS= service file setting now additionally mounts a tmpfs
+          on /run if it is not already a mount point. A writable /run has always
+          been a requirement for a functioning system, but this was not
+          guaranteed when using a read-only image.
+          Users can always specify BindPaths or InaccessiblePaths as overrides,
```

Comment:
well, bingo, please suffix with `=`.

Reply:
eheh got one out of three right, it's something :-D